### ISSUE TITLE
Use a show-hide component for the copyright information

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#170]
### Problem
[molecules/molecules.image](https://a-ui.github.io/core_branding_scss/#molecules-image)

The copyright information is shown on hover and cannot be accessed with a speech command or the keyboard.

```html
<figure>
  <div class="m-image">
    <img src="https://placeimg.com/1000/500/tech" alt="figure image">
    <div class="m-image__copyright a-copyright" aria-haspopup="true">
      <div class="a-copyright__sign">
        <span class="fa fa-copyright"></span>
      </div>
      <span class="a-copyright__label">Lorem Pixel</span>
    </div>
  </div>
  <figcaption>Some fancy caption about this photo</figcaption>
</figure>
```

### Solution

* Delete the aria-haspopup attribute. It can only be used in combination with an ARIA role menu, listbox, tree, grid, or dialog,
* Use a button with the aria-expanded attribute (show/hide component).

Suggestion:

```html
<figure class="m-image">
  <img src="https://placeimg.com/1000/500/tech" alt="figure image">
  <figcaption class="m-image__copyright a-copyright">
    <button class="a-copyright__sign" aria-expanded="true">
      <span class="fa fa-copyright">Copyright </span>
    </button>
    <span class="a-copyright__label">Lorem Pixel </span>
    <span>Some fancy caption about this photo</span>
  </figcaption>
</figure>
```

### Links and articles

* [ARIA Design pattern and widgets: Disclosure (Show/Hide)](https://www.w3.org/TR/wai-aria-practices-1.2/#disclosure)
* [Accessible Rich Internet Applications (WAI-ARIA) 1.2: aria-expanded (state)](https://www.w3.org/TR/wai-aria-1.2/#aria-expanded)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)
